### DISCUSSION TITLE
Installation procedure outdated

### DISCUSSION THREAD

**Discussion by virtualjoe** [fsaris/home-assistant-zonneplan-one#106] [General]
The information mentioned in the Installation instructions seems to be outdated:

----

Install with HACS (recommended)

Do you have HACS installed?

Search integrations for Zonneplan ONE
Click _Install_ **<-- This is now "Download"**
Restart Home Assistant

Setup

In Home Assistant click on _Configuration_  **<-- Does not exist anymore, Instead goto HACS in sidebar**
Click on Integrations
Click on + Add integration
Search for and select Zonneplan ONE
Enter _you emailaddress_ you also use in the Zonneplan ONE app **<-- There is no option to enter Email address (issue?)**
You will get an email to verify the login.
Click "Save"
Enjoy

----

Tested installation on HA/HACS version:
Core 2024.4.2
Supervisor 2024.04.0
Operating System 12.1
Frontend 20240404.1
HACS Integration version: | 1.34.0
HACS Frontend version: | 20220906112053

**Comment by fsaris** [comment]
Looks that you are mixing some parts.

1. Install the custom component using HACS
2. Add the integration [![Open your Home Assistant instance and start setting up a new integration for Zonneplan.](https://my.home-assistant.io/badges/brand.svg)](https://my.home-assistant.io/redirect/brand/?brand=zonneplan_one)

Step 1 is just to make the integration available in your Home Assistant installation.

Step 2 is adding/configuring the integration. Like any core integration
